multi_partner_learning.py

```diff
@@ -476,21 +504,49 @@ def init_with_agg_models(self):
         return partners_model_list
 
     @staticmethod
-    def collaborative_round_fit(model_to_fit, train_data, val_data, batch_size):
+    def fit_model(model_to_fit, train_data, val_data, batch_size):
         """Fit the model with arguments passed as parameters and returns the history object"""
 
         x_train, y_train = train_data
-        history = model_to_fit.fit(
-            x_train,
-            y_train,
-            batch_size=batch_size,
-            epochs=1,
-            verbose=0,
-            validation_data=val_data,
-        )
-
+        if isinstance(model_to_fit, type(LogisticRegression())):
+            history = model_to_fit.fit(x_train, y_train)
+            [loss, acc] = MultiPartnerLearning.evaluate_model(model_to_fit, train_data)
+            [val_loss, val_acc] = MultiPartnerLearning.evaluate_model(model_to_fit, val_data)
```

Comment:
Why use this syntax with the name of the class here?

Reply:
Because of the `@staticmethod` tag.

Is there any way to do better ?

Reply:
Ok then. We could question the benefit of making it a staticmethod in the first place, but let's not do that we have enough on our plate :smiley: